transmogrifier/helpers.py

```diff
@@ -75,31 +75,6 @@ def parse_date_from_string(
     return None
 
 
-def parse_geodata_string(geodata_string: str, source_record_id: str) -> list[float]:
```

Comment:
Since we're just taking the WKT string as is given than OpenSearch can handle it, we don't need this function to do the parsing that we originally though was needed. Does that make sense?